plugins/plugin-chart-echarts/src/Timeseries/EchartsTimeseries.tsx

```diff
@@ -85,10 +90,41 @@ export default function EchartsTimeseries({
     mouseout: () => {
       currentSeries.name = '';
     },
+    legendselectchanged: payload => {
+      const currentTime = Date.now();
+      // 300 is the interval between two legendselectchanged event
+      if (currentTime - lastTimeRef.current < 300 && lastSelectedLegend.current === payload.name) {
```

Comment:
Since we can't pass click events (both click and double-click) to legend to drive the series change, we listen to the `legendselectchanged`  and the time interval to simulate the double-click event.  The value of 300 can be adjusted if user think is so fast.